lib/reports/time/ReportTimeUtils.js

```diff
@@ -0,0 +1,191 @@
+/**
```

Comment:
we should at least add some unit tests for these, even if we're not going to write a side-by-side test yet.  (maybe we can add that to the issue, and write them up when we circle back to it?)